### ISSUE TITLE
Windows Narrator says that the Checkbox selection state has changed when visually the checkbox state doesn't change.

### ISSUE THREAD

**Issue by davidprejban** [OfficeDev/office-ui-fabric-react#5425]
### Bug Report 
- __Package version(s)__: 
6.21.0

- __Browser and OS versions__:
Windows server 2016: Version 1607 OS Build 1493.2339
Internet Explorer 11: Version 11.2239.14393.0
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix?
Yes
 
Requested priority:
High
 
#### Describe the issue:
Windows narrator will not say whether or not the color is selected when selecting the color.
 
Reproduction steps:
1. Open I.E 11
2. Navigate to:
  a. https://developer.microsoft.com/en-us/fabric#/components/checkbox
3. Turn on windows narrator
  a. https://support.microsoft.com/en-us/help/22817
4. Tab until you have a checkbox example selected
5. Press 'caps' + 'enter'

#### Actual behavior:
Checkbox does not visibly change state while narrator says the state has changed.
E.g. 
1. Checkbox shows as checked.
2. Press 'caps' + 'enter'
3. Narrator says "unchecked"
4. Checkbox shows as checked.
 
#### Expected behavior:
Checkbox shows what narrator says the state is "checked" or "unchecked" and checkbox shows as checked or unchecked the same as narrator says it is.

**Comment by dzearing** [comment]
is this a narrator bug?

**Comment by betrue-final-final** [comment]
I repro'd the bug in IE11. Note that you have to be in scan mode. Checkbox doesn't seem to be working at all in Edge with Narrator. Neither space nor enter will select the standard checkbox.

**Comment by JasonGore** [comment]
My findings are almost completely the opposite of Ben's. Edge + Narrator works fine for me with scan mode off. 

IE11 (only when using Caps + Enter) exhibits the behavior reported. However, Space and Enter work fine. (Ctrl + Enter works fine when Narrator is disabled.)

It turns out IE11 also exhibits this behavior with [an ARIA reference example](http://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/checkbox/checkbox-1.html).

Further investigation reveals that Checkbox's `button` element's `onClick` handler never gets called when using Ctrl + Enter in IE11 with Narrator enabled. I'm not sure whether this translates to a Narrator issue or an IE11 issue, but seeing as the lowest level handler isn't getting called, I don't see how we could fix this in Fabric.

**Comment by betrue-final-final** [comment]
Ok, now it's working fine. This is worrisome as it didn't work before.

**Comment by betrue-final-final** [comment]
Would you like me to reiterate that I no longer repro the issue? It would make sense to leave the thread as a fully transparent process...I did repro the bug, but later on I tried it and it didn't repro after you commented.

For clarity I'll add this
**I no longer repro this bug**

**Comment by JasonGore** [comment]
The issue here is that the reported issue is specifically regarding `CAPS + Enter`, but your successive comments have expanded scope beyond what was reported, and then you made a blanket statement that the issue no longer occurs. 

The CAPS + Enter issue still occurs. It is not right to say it no longer occurs.

Any other behavior you see should be reported in another issue.

This is all to minimize confusion against what was originally reported.

**Comment by betrue-final-final** [comment]
Apologies, Jason is correct. I incorrectly said that the bug no longer repros, but actually it does, just not in Edge, which would have been a separate bug. To add further detail:

**This bug repros in IE11**. Both CAPS + Enter AND Space cause Narrator to say the state change, but the visual does not switch.

**Comment by JasonGore** [comment]
This has been acknowledged and reported internally as an IE11 issue. As this is not Fabric-related, I will close this issue.
